### DISCUSSION TITLE
Load build files from path in disk

### DISCUSSION THREAD

**Discussion by Mir4nda** [jeffreylanters/react-unity-webgl#438] [Question and Answer] [answered]
Can I, for example, load the build files from disk with a static path (like c://buildWegbl)?

See this diagram please: https://i.imgur.com/JtDEqzL.jpg

**Comment by jeffreylanters** [comment, marked as answer]
Hi! It has nothing to do with the capabilities of the module, but rather with these of the browser. I'm not sure if any major browser is allowed to just pull a random file at a given path from the user's device, since this would introduce a major security flaw.

What are you trying to accomplish, I might be able to help?

**Reply by Mir4nda** [reply]
Yes I know :/ It was in an attempt to get around my problem....

I can load the webgl build on the windows server ( IIS ) and I've had a lot of configurations in an attempt to solve the cache, so I have to download the public.data file just once but it's not happening and so I thought if I could load it webgl from the client side....which seems absurd but I'm all for it :)
